Sources/FirestoreManager/FirestoreCRUD/Fetch.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  FirestoreManager+Fetch.swift
+//  
+//
+//  Created by Anna Münster on 26.08.21.
+//
+
+import Foundation
+import FirebaseFirestore
+
+extension FirestoreManager {
+    /**
+     Fetch and listen to a collection from Firebase.
+     
+     - Parameter reference: The collection name
+     - Parameter userFilter: Filter all objects by the current user's id
+     - Parameter filters: Dictionary of the filter key and the value
+     - Parameter orderBy: Key to order objects by
+     - Parameter descending: Whether orderBy key should descend
+     - Parameter limit: Limit number of fetched items
+     */
+    public static func fetchCollection<T>(_ reference: ReferenceProtocol,
+                                   userFilter: Bool = true,
```

Comment:
The userFilter isn't called within the function. Either we need to call it or remove it. If we use it, there needs to be the property for the userID as well.

Reply:
I removed it in here and will provide a new method signature probably in the next PR